src/app/home/avatar-generator/selector/atribute-selector.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit, Input, Inject } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import {AvatarDecoratorItem, AvatarOptions} from '../Avatartypes';
+import {buildAvatar} from '../AvatarUtils';
+import {getPreview} from '../AvatarSelectorPath';
+
+@Component({
+  // tslint:disable-next-line: component-selector
+  selector: 'atribute-selector',
```

Comment:
Please add the app prefix to the selector of this component: `wgj-attribute-selector`
Also, there is a small typo here, atribute should be attribute

Reply:
@AllanLeon can you share a link why is using prefix selector? :D

Reply:
Sure :)
Using the prefix prevents element name collisions with other components or native html elements and makes it easier to differentiate the components that belong to this app.
https://angular.io/guide/styleguide#style-02-07